.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### **Describe the bug:**
+A clear and concise description of what the bug is.
+
+### **To Reproduce:**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
Just mention, please provide the steps to reproduce the error. 
As this is not UI app at present, these steps may ceate a confusion to user.

Reply:
Ok I'll change